config/main.py

```diff
@@ -490,6 +490,15 @@ def remove_portchannel(ctx, portchannel_name):
     db = ctx.obj['db']
     db.set_entry('PORTCHANNEL', portchannel_name, None)
 
+@portchannel.command('mtu')
+@click.argument('portchannel_name', metavar='<portchannel_name>', required=True)
+@click.argument('mtu', metavar='<mtu>', required=True)
+@click.pass_context
+def set_portchannel_mtu(ctx, portchannel_name, mtu):
+    """set port channel mtu"""
+    db = ctx.obj['db']
+    db.mod_entry('PORTCHANNEL', portchannel_name, {"mtu":mtu})
```

Comment:
I see that almost all port channel related Cmds do not check much on the given parameter(s).  I really think we should fix all previous Portchannel related commands to at least validate that the given parameter(s) make sense.  For new PortChannel command being added should start implementing parameter checking as a basic requirment.  For this particular case it should check that the specified portchannel exists in the Config DB and if not, reject the request and print something so that the user is aware that he has made a mistake and can try again with the correct parameter...

Reply:
Currently the port MTU is being configured the following way:
    config interface mtu Ethernet0 5000
This is handled via portconfig.py
I browsed that code and it seems to be enhanced to handle Portchannel by checking the PORTCHANNEL in the config DB if the interface specified is LAG. so something as following can be done:
    config interface mtu PortChannel0002 5000
I feel this may be a better approach instead.  This also make the MTU config with consisten CLI format.  portconfig.py has included basic parameter checks for the interface and display error correctly.  So I would suggest that we handle this by changing portconfig.py instead.